src/pages/CreateProposal/ProposalEditor.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from 'react'
+import styled from 'styled-components'
+import { ResizingTextArea, TextInput } from 'components/TextInput'
+import { Trans } from '@lingui/macro'
+
+const ProposalEditorHeader = styled.div`
+  font-size: 14px;
+  font-weight: 500;
+  color: ${({ theme }) => theme.text2};
+`
+
+const ProposalTitle = memo(styled(TextInput)`
+  margin-top: 7.5px;
+  margin-bottom: 7.5px;
+`)
+
+const _ProposalEditor = ({
+  className,
+  title,
+  body,
+  onTitleInput,
+  onBodyInput,
+}: {
+  className?: string
+  title: string
+  body: string
+  onTitleInput: (title: string) => void
+  onBodyInput: (body: string) => void
+}) => {
+  const bodyPlaceholder = `## Summary
```

Comment:
not to scope creep but did you consider adding a markdown preview?

Reply:
Yeap, I think it got p2-ed for now.